src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
+        <td class="numeric">3.93%</td>
+        <td class="numeric">3.73%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="Percent of pages having each viewport meta tag <code>content</code> attribute value.",
+      sheets_gid="479500659",
+      sql_file="../03_Markup/summary_pages_by_device_and_viewport.sql"
+    ) }}
+  </figcaption>
 </figure>
 
-#### CSS Media Queries
-
-Media queries are a CSS3 feature that play a fundamental role in responsive web design as they allow you to specify conditions to apply styling only when the browser and device match certain rules. This allows you to create different layouts for the same HTML depending on the viewport size.
-
-When analyzing their use it was found that 80.29% of desktop homepages and 82.92% of the mobile ones are using either a height, width or aspect-ratio CSS feature, meaning that a high percentage of pages have responsive features. The most popularly used features can be seen in the table below.
-
-<figure markdown>
-Feature | Mobile | Desktop
--- | -- | --
-`max-width` | 78.98% | 78.33%
-`min-width` | 75.04% | 73.75%
-`-webkit-min-device-pixel-ratio` | 44.63% | 38.78%
-`orientation` | 33.48% | 33.49%
-`max-device-width` | 26.23% | 28.15%
-
-<figcaption>{{ figure_link(caption="Media query usage.", sheets_gid="1141218471", sql_file="TODO.sql") }}</figcaption>
+#### CSS media queries
+
+Media queries are a CSS3 feature that play a fundamental role in responsive web design, as they allow you to specify conditions to apply styling only when the browser and device match certain rules. This allows you to create different layouts for the same HTML depending on the viewport size.
+
+We found that 80.29% of desktop home pages and 82.92% of the mobile ones are using either a `height`, `width`, or `aspect-ratio` CSS feature, meaning that a high percentage of pages have responsive features. The most popularly used features can be seen in the table below.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Feature</th>
+        <th>Mobile</th>
+        <th>Desktop</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>max-width</code></td>
+        <td>78.98%</td>
+        <td>78.33%</td>
+      </tr>
+      <tr>
+        <td><code>min-width</code></td>
+        <td>75.04%</td>
+        <td>73.75%</td>
+      </tr>
+      <tr>
+        <td><code>-webkit-min-device-pixel-ratio</code></td>
+        <td>44.63%</td>
+        <td>38.78%</td>
+      </tr>
+      <tr>
+        <td><code>orientation</code></td>
+        <td>33.48%</td>
+        <td>33.49%</td>
+      </tr>
+      <tr>
+        <td><code>max-device-width</code></td>
+        <td>26.23%</td>
+        <td>28.15%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="Percent of pages that include each media query feature.",
+      sheets_gid="1141218471",
+      sql_file="TODO..sql"
```

Comment:
@Tiggerito I think this change obsoleted the last comment thread, reviving it here

Reply:
My response was:

It looks like I stole this from you 😮

I asked the CSS chapter for some help:

#898 (comment)

And you provided the data...

https://docs.google.com/spreadsheets/d/1sMWXWjMujqfAREYxNbG_t1fOJKYCA6ASLwtz4pBQVTw/edit#gid=1374950017